mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/composeScreen/ItemCasualkList.kt

```diff
@@ -0,0 +1,102 @@
+package org.mifos.mobilewallet.mifospay.bank.composeScreen
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.mifos.mobilewallet.model.domain.BankAccountDetails
+import org.mifos.mobilewallet.mifospay.R
+
+@Composable
+fun ItemCausualList(
```

Comment:
ambiguous naming convention. what is Causual

Reply:
Write something related to feature

Reply:
changed ListItemWithImage ..done